### PR TITLE
Add AWS CLI flavour for Batch tutorial

### PR DESCRIPTION
**Pull request by ciaranevans**
This PR adds AWS CLI commands as alternatives to using the console when following the AWS Batch tutorial

It also tidies up some code block highlighting in the EC2 tutorial

### CODE REVIEW

**Review by Lawouach — DISMISSED** [review]
LGTM

File: sources/reference/tutorials/batch.md [hunk 1]
```diff
@@ -226,33 +252,85 @@ latest: digest: sha256:9702b9cf63a6e4961689a661340fc0573d28d0e7f506b90fa5d080e4e
...
+    in `Networking`
+        * For this guide, we'll use the values AWS filled in
+    * Add a tag - We recommend at minimum, adding a tag `{"OWNER": "your-name"}`
+    * Select `Create compute environment`
+
+=== "AWS CLI"
+
+    * To create your Compute environment, you'll need to know the subnets you'll
+    provide it, take note of the output for the following command, note down the
+    subnet IDs you want to use, replacing `VPC_ID` with the VPC ID you're deploying
```
Comment (CharlieMoon37) [inline sources/reference/tutorials/batch.md:273]: Is it possible to auto-fill the values that AWS would fill in if we were doing it manually: https://github.com/chaostoolkit/chaostoolkit-documentation/pull/142/files#diff-14961ade072f5656a60b94a37b75aaa8d80bb0ff25da189d429f39cd38e92dbfR265
If not, might we need instructions on how to retrieve that value?

Comment (ciaranevans) [reply sources/reference/tutorials/batch.md:273]: Which value do you mean?

If you mean the subnets, that's just what the command below this text is doing, listing the subnets

Comment (CharlieMoon37) [reply sources/reference/tutorials/batch.md:273]: I am meaning the `VPC_ID`
To access the default one, I went to create one manually on AWS and used the id it gave me to replace "VPC_ID"

Comment (ciaranevans) [reply sources/reference/tutorials/batch.md:273]: So we could show them how to list VPCs, but for your case, the default VPC is the only vpc available in the account

Comment (CharlieMoon37) [reply sources/reference/tutorials/batch.md:273]: Sounds good and yes I understand that!
I just wasn't sure how I would retrieve the VPC_ID to initiate that command in the first place
Are we just expecting the users to have theirs easily accessible?

Comment (ciaranevans) [reply sources/reference/tutorials/batch.md]: Big assumption is someone will have the AWS console up/know how to query for this, but I've added a step

Comment (ciaranevans) [reply sources/reference/tutorials/batch.md]: We have to remember that we're not trying to teach folks AWS, just how to setup CTK on AWS, which admittedly is a fine line

Comment (CharlieMoon37) [reply sources/reference/tutorials/batch.md]: Makes sense and looks good!

File: sources/reference/tutorials/batch.md [hunk 2]
```diff
@@ -287,73 +412,192 @@ By creating a job role for our Job, we can:
...
+            "assignPublicIp": "ENABLED"
+        },
+        "fargatePlatformConfiguration": {
+            "platformVersion": "LATEST"
+        }
+    }
+    ```
+
+    * Replace `ECR_IMAGE_URI` in the file with the URI of the image you pushed
+    to ECR. Replace `JOB_ROLE_ARN` with the ARN of `ctk-batch-job-role`. Replace
```
Comment (CharlieMoon37) [inline sources/reference/tutorials/batch.md:582]: Can we replace `JOB_ROLE_ARN` with `arn:aws:iam::<your-aws-account-id>:role/ctk-batch-job-role` to simplify things a bit?

File: sources/reference/tutorials/batch.md [hunk 3]
```diff
@@ -287,73 +412,192 @@ By creating a job role for our Job, we can:
...
+    `EXECUTION_ROLE_ARN` with the ARN of `ctk-batch-execution-role`.
```
Comment (CharlieMoon37) [inline sources/reference/tutorials/batch.md:583]: Similarly, can we replace `EXECUTION_ROLE_ARN` with `arn:aws:iam::<your-aws-account-id>:role/ctk-batch-execution-role`

File: sources/reference/tutorials/batch.md (hunk 2 above)
Comment (ciaranevans) [reply sources/reference/tutorials/batch.md]: Yep!

File: sources/reference/tutorials/batch.md (hunk 3 above)
Comment (ciaranevans) [reply sources/reference/tutorials/batch.md]: Yep, done

File: sources/reference/tutorials/batch.md [hunk 4]
```diff
@@ -367,43 +610,98 @@ Now that you have:
...
+    [2021-08-19 14:02:43 INFO] Steady state hypothesis is met!
+    [2021-08-19 14:02:43 INFO] Let's rollback...
+    [2021-08-19 14:02:43 INFO] No declared rollbacks, let's move on.
+    [2021-08-19 14:02:43 INFO] Experiment ended with status: completed
+    No newer events at this moment. Auto retry paused. Resume
+    ```
+
+=== "AWS CLI"
+
+    * Run your Job, replacing YOUR_NAME with your name, take note of `jobId` in
```
Comment (CharlieMoon37) [inline sources/reference/tutorials/batch.md:655]: replacing \`YOUR_NAME\` with your name

Comment (ciaranevans) [reply sources/reference/tutorials/batch.md]: done

**Review by CharlieMoon37 — APPROVED** [review]
Absolutely Brilliant!

File: sources/reference/tutorials/batch.md [hunk 5]
```diff
@@ -62,20 +62,35 @@ our SSH to ensure that the EC2 instance is in a `running` state.
...
-* Select the Subnet to deploy into via the `Subnet` dropdown
-* Go onto `Add Storage` - For now, the defaults will be fine
-* Go onto `Add Tags` - We recommend at minimum, adding a tag `{"OWNER":
-"your-name"}`
-* Go onto `Configure Security Group`
-* Click the `X` to the right of the SSH rule, you won't need this
-* Go onto `Review and Launch` - Select `Launch`
-* Select `Proceed without a key pair`, check the tickbox, and click `Launch
-Instances`
+=== "AWS Console"
```
Comment (CharlieMoon37) [inline sources/reference/tutorials/batch.md:65]:
```suggestion
=== "AWS Console For Nerds"
```

Comment (ciaranevans) [reply sources/reference/tutorials/batch.md:65]: You cheeky wotsit.

**Pull request merged**